typescript/ecs/ec2-service-with-alb/index.ts

```diff
@@ -0,0 +1,57 @@
+import ecs = require('@aws-cdk/aws-ecs');
```

Comment:
Maybe call this advanced-alb-configuration?

The advantage of going this route over the simple LoadBalancedXxxService should be obvious.

Reply:
is `ec2-service-with-advanced-alb-config` too much of a mouthful?

Reply:
No that's fine